frontend/src/App.tsx

```diff
@@ -201,6 +210,11 @@ async function GameController(initData: TownJoinResponse,
 
 function App(props: { setOnDisconnect: Dispatch<SetStateAction<Callback | undefined>> }) {
   const [appState, dispatchAppUpdate] = useReducer(appStateReducer, defaultAppState());
+  const [instructions, setShowInstructions] = useState<boolean>(appState.showInstructions);
```

Comment:
If we have `showInstructions` as part of the `appState`, do we still need this separate useState for `showInstructions`, or should we be modifying `appState` instead?

Reply:
Good point, I've implemented this now!